#goal
geometry_msgs/PoseArray InputPoint
string[] InitLabel
---
#result
geometry_msgs/PoseArray[] FinalTrajectories
string[] FinalLabel
---
#feedback
string output